sdk/highlight-react/src/components/ErrorBoundary.tsx

```diff
@@ -71,7 +73,7 @@ export class ErrorBoundary extends React.Component<
 		if (onError) {
 			onError(error, errorInfo.componentStack)
 		}
-		if (showDialog) {
+		if (!noShowDialog) {
```

Comment:
Not a huge fan of prefixing a prop with "no".

What if we changed the default value of showDialog to true and if someone wanted to opt out, they could set it to false.

I get that would be a breaking change but I tend to think this is already a breaking change since the code is no longer referencing showDialog.

Reply:
that's a great point, should have thought of that :man_facepalming: 
updated!

showDialog=true
![image](https://user-images.githubusercontent.com/1351531/235020550-799e9c2f-0c4a-4f8d-ba8c-57783980453f.png)

showDialog=false
![image](https://user-images.githubusercontent.com/1351531/235020942-5d2b1bea-91e8-44a4-b248-df375ef1c7b9.png)

showDialog undefined
![image](https://user-images.githubusercontent.com/1351531/235022044-e26b4b86-9a81-4064-912a-c914bccd7b22.png)